config/locales/fra/action.yml

```diff
@@ -1097,10 +1103,18 @@ fra:
       filter_on_attribute_of_class: "Filtre sur %{attribute} (%{class})"
       special-subscriber: "Filtre Abonné"
     configure: "Configurer"
+    confirmation: "Confirmation"
     confirmed: "Confirmées" #?
     confirmed_at: "Confirmée le" #?
     connect: "Se connecter"
     consult: "Consulter" #?
+    consult_the_day_book:
+      zero: "Consulter le brouillard (%{count} écriture)"
+      one: "Consulter le brouillard (%{count} écriture)" #?
+      other: "Consulter le brouillard (%{count} écritures)" #?
+    consult_unbalanced_entries:
+      one: "Consulter les écritures à équilibrer (%{count})" #?
```

Comment:
Singular case : Change with "Consulter l'écriture ..."

Reply:
Following the design
![screenshot-2018-7-31 accountancy 2018 mockups sales](https://user-images.githubusercontent.com/26605504/43461879-e2215aa8-94d4-11e8-9d19-cf531580966d.png)

Reply:
I was talking about line 1116 : `one: "Consulter les écritures à équilibrer (%{count})"`

Reply:
My bad